src/views/core/members/ListUnconfirmed.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <div class="tile is-ancestor">
+    <div class="tile is-parent is-vertical">
+      <article class="tile is-child">
+        <h4 class="title">Unconfirmed user list</h4>
+        <div class="field">
+          <label class="label">Search by name or surname</label>
+          <div class="control">
+            <input class="input" type="text" data-cy="query" v-model="query" placeholder="Search by name or surname" @input="refetch()">
+          </div>
+        </div>
+
+        <b-table
+          :data="users"
+          :loading="isLoading"
+          paginated
+          backend-pagination
+          backend-sorting
+          :total="total"
+          :per-page="limit"
+          @page-change="onPageChange"
+          :default-sort="[sortField, sortOrder]"
+          @sort="onSort">
+          <template slot-scope="props">
+            <b-table-column field="id" label="#" numeric sortable>
+              {{ props.row.id }}
+            </b-table-column>
+
+            <b-table-column field="first_name" label="Name and surname" sortable width="150">
+              <router-link :to="{ name: 'oms.members.view', params: { id: props.row.username || props.row.id } }">
+                {{ props.row.first_name }} {{ props.row.last_name }}
+              </router-link>
+            </b-table-column>
+
+            <b-table-column field="email" label="Email">
+              {{ props.row.email }}
+            </b-table-column>
+
+            <b-table-column field="date_of_birth" label="Birthday" sortable width="150">
```

Comment:
I don't think we need the birthday, address and about me fields here

Reply:
makes sense, deleted these fields